Traceback (most recent call last):
  File "gmm_run_concrete_waveform.py", line 106, in <module>
    tmp_flow,tmp_loss=trainGMMRealNVP(
  File "/arc/project/st-tdjc-1/madmix/examples/gmm/waveform/../../../src/concrete.py", line 704, in trainGMMRealNVP
    loss = -flow.log_prob(sample).mean()
  File "/arc/project/st-tdjc-1/madmix/examples/gmm/waveform/../../../src/concrete.py", line 406, in log_prob
    return self.prior.log_prob(z) + logp
  File "/arc/project/st-tdjc-1/madmix/examples/gmm/waveform/../../../src/concrete.py", line 558, in log_prob
    xc_lp += self.invwis.log_prob(torch.moveaxis(Sigmas[k,...],2,0))
  File "/home/giank/.local/lib/python3.8/site-packages/torch/distributions/wishart.py", line 271, in log_prob
    self._validate_sample(value)
  File "/home/giank/.local/lib/python3.8/site-packages/torch/distributions/distribution.py", line 300, in _validate_sample
    raise ValueError(
ValueError: Expected value argument (Tensor of shape (1000, 2, 2)) to be within the support (PositiveDefinite()) of the distribution Wishart(covariance_matrix: torch.Size([2, 2]), df: 100.0), but found invalid values:
tensor([[[3.1258e+00, 4.8595e+01],
         [4.8595e+01, 7.5548e+02]],

        [[2.4518e+00, 4.6431e+01],
         [4.6431e+01, 8.7927e+02]],

        [[6.7329e+00, 6.2552e+01],
         [6.2552e+01, 5.8113e+02]],

        ...,

        [[2.0997e+00, 3.2238e+01],
         [3.2238e+01, 4.9498e+02]],

        [[1.5715e+01, 2.1418e+02],
         [2.1418e+02, 2.9190e+03]],

        [[1.3510e+01, 9.6468e+01],
         [9.6468e+01, 6.8885e+02]]], grad_fn=<PermuteBackward0>)
